### DISCUSSION TITLE
Current situation of this project

### DISCUSSION THREAD

**Discussion by duszmox** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#126] [General]
Hi everyone,
Today I have some bad news. 
First of all, @mhl221135 is leaving this project because he sold his laptop. I would like to thank him for his contribution and help.
Secondly, a week ago my laptop decided to die, something got shorted out on the motherboard or something similar, because it would post at all, nothing shows up on the screen, and currently I don't have enough money to buy a new one, because I'm still a high school student. But thankfully over the course of the past year I managed to convince a few of my close friends to buy the same model of Thinkpad as I have and install macOS on it as well, so **I can continue this project thankfully**. The only downside is that I cannot release the EFI folder on the same day as they release the new OpenCore updates (every first Monday of the month), so expect them around Wednesday to Friday. 

Lastly, I released the 0.7.3 EFI folder today as a pre-release, it isn't tested yet, so be careful! If you test is out, and it works for you, please let me know!

Thank you for understanding the situation, and also thank you for hitting over 200 stars, now we are top ten on github in the category of Hackintosh laptops.

**Comment by simprecicchiani** [comment]
The commitment you and @mhl221135 have put on the project is impressive. I'm sorry to hear the bad news, it's so unfortunate. Keep up the good work guys!

**Comment by webdeveloper1877** [comment]
hi, I noticed that the new efi 0.7.3 has a problem with the adjustment of the brightness in the menubar the regulator is not active. I went back to 0.7.2

**Reply by duszmox** [reply]
Which version of macOS are you running?

**Comment by truongmaixuan** [comment]
Thank you for all. I have an issue that I can't boot to macOS when I try to update from Open core 0.7.2 to 0.7.3 (also 0.7.4).
I'm using macOS 12 beta 7. 
Could you give me any advice?
